examples/prompty/chat-basic/chat-with-prompty.ipynb

```diff
@@ -260,17 +305,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "eval_run = pf.run(\n",
-    "    flow=eval_prompty,\n",
-    "    data=\"./data.jsonl\",  # path to the data file\n",
-    "    run=base_run,  # specify base_run as the run you want to evaluate\n",
-    "    column_mapping={\n",
-    "        \"messages\": \"${data.chat_history}\",\n",
-    "        \"question\": \"${data.question}\",\n",
-    "        \"answer\": \"${run.outputs.answer}\",\n",
-    "    },\n",
-    "    stream=True,\n",
-    ")"
+    "# eval_run = pf.run(\n",
```

Comment:
we should remove the section if it doesn't work? @0mza987

Reply:
and it works on my machine?
![image](https://github.com/microsoft/promptflow/assets/37076709/0f688b49-b8e0-49e3-b9e5-d1ed1bcb8946)

Reply:
offline synced.